spec/features/braintree_credit_card_checkout_spec.rb

```diff
@@ -48,14 +48,20 @@
       within_frame("braintree-hosted-field-cvv") do
         fill_in("cvv", with: "123")
       end
-
       click_button("Save and Continue")
       within("#order_details") do
         expect(page).to have_content("CONFIRM")
       end
       click_button("Place Order")
+    end
+
+    it "checks out successfully" do
       expect(page).to have_content("Your order has been processed successfully")
     end
+
+    it "sets the payment type of source to CreditCard" do
+      expect(SolidusPaypalBraintree::Source.last.payment_type).to eq('CreditCard')
```

Comment:
This assertion does not belong in feature spec. Could we do this at the unit test level? We should probably also avoid using last unless we're sure the system is only creating the one record.